perception/autoware_lidar_transfusion/lib/network/network_trt.cpp

```diff
@@ -23,6 +23,23 @@
 namespace autoware::lidar_transfusion
 {
 
+inline NetworkIO nameToNetworkIO(const char * name)
+{
+  static const std::unordered_map<std::string_view, NetworkIO> name_to_enum = {
+    {"voxels", NetworkIO::voxels},
+    {"num_points", NetworkIO::num_points},
+    {"coors", NetworkIO::coors},
+    {"cls_score0", NetworkIO::cls_score},
+    {"dir_cls_pred0", NetworkIO::dir_pred},
+    {"bbox_pred0", NetworkIO::bbox_pred}};
```

Comment:
Please, switch `dir_cls_pred0` with `bbox_pred0` in line 33 & 34 and order of [enum](https://github.com/autowarefoundation/autoware.universe/blob/f256c0bed26e8b26b1eab39b8085a5c6c0be3722/perception/autoware_lidar_transfusion/include/autoware/lidar_transfusion/utils.hpp#L39) as well.
This change is only for readability purposes, I want to stick our enum with [original implementation](https://github.com/open-mmlab/mmdeploy/blob/main/mmdeploy/codebase/mmdet3d/models/mvx_two_stage.py#L99).

Reply:
agreed. Fixed.

Reply:
Style: there is one unnecessary comma at the end after your change, could you remove it and bring back one enum per line? Maybe that is why pre-commit did this weird style fix.

Reply:
I removed the comma, but pre-commit still uses the wierd style fix